django/views/debug.py

```diff
@@ -70,6 +70,14 @@ def get_safe_settings():
     return settings_dict
 
 
+def get_safe_request_meta(request):
+    "Returns a dictionary of request.META with sensitive keys blurred out."
```

Comment:
Returns -> Return (there's another ticket to fix existing docstrings)
"sensitive keys" -> "values of sensitive keys"?